var/spack/repos/builtin/packages/cctools/package.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import llnl.util.tty as tty
+
+class Cctools(AutotoolsPackage):
+    """The Cooperative Computing Tools (cctools) enable large scale
+       distributed computations to harness hundreds to thousands of
+       machines from clusters, clouds, and grids.
+    """
+
+    homepage = "https://github.com/cooperative-computing-lab/cctools"
+    url      = "https://github.com/cooperative-computing-lab/cctools/archive/release/6.1.1.tar.gz"
+
+    version('6.1.1', '9b43cdb3aceebddc1608c77184590619')
+
+    depends_on('openssl')
+    depends_on('perl+useshrplib', type=('build','run'))
+    depends_on('python@:3', type=('build','run'))
```

Comment:
I did do that intentionally.

It has something funny going on that I haven't yet grokked (handcoded configure script).  It almost seems that you can configure it to use both simultaneously (`--with-python-path` and `--with-python3-path`).

I know it works as is and have that bit of poking around on my TODO list.